src/formatter/json_formatter.ts

```diff
@@ -293,7 +293,11 @@ export default class JsonFormatter extends Formatter {
     }
     if (testStepAttachments?.length > 0) {
       data.embeddings = testStepAttachments.map((attachment) => ({
-        data: attachment.body,
+        data:
+          attachment.contentEncoding ===
+          messages.AttachmentContentEncoding.IDENTITY
+            ? Buffer.from(attachment.body).toString('base64')
```

Comment:
Is browser compatibility a requirement for this code? (`Buffer` is node)

Reply:
No, we're Node.js only at the moment so `Buffer` is fine.